flask_resty/view.py

```diff
@@ -407,6 +732,14 @@ def commit(self):
             raise self.resolve_integrity_error(e)
 
     def resolve_integrity_error(self, error):
+        """Resolves the `error`. If the postgres error code corresponds to a
```

Comment:
interesting, flask and sqlalchemy support multiple DBs. is this method coupled with postgres?

Reply:
I guess this would be a no op if anything is used other than postgres /shrug